spring-boot/src/main/java/org/springframework/boot/SpringApplication.java

```diff
@@ -843,6 +845,17 @@ public void setEnvironment(ConfigurableEnvironment environment) {
 	}
 
 	/**
+	 +	 * Sets the environment variables prefix that will be used to filter those variables
+	 +	 * for this application.
+	 +	 * @param environmentPrefix the prefix
+	 +	 */
+	public void setEnvironmentPrefix(String environmentPrefix) {
```

Comment:
@snicoll I'm not sure if this is the best way to configure the environment prefix. I'd like some opinions on that.